modules/ROOT/pages/key-configurations.adoc

```diff
@@ -0,0 +1,11 @@
+= Configuring Behavior during Runtime
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: apikit-4-tasks-index.adoc
+
+In contrast to newer modules created using the Java SDK, APIkit for REST has distinctive aspects in its configuration and behavior that warrant a more comprehensive explanation. This section presents relevant settings that determine the behavior of the module during runtime.
```

Comment:
Is all this necessary? 

Can you just say something like:
The following configuration settings determine the behavior of the APIkit for REST module during runtime:

Reply:
You're right! Thanks!